core/iwasm/fast-jit/jit_ir.c

```diff
@@ -1425,3 +1425,16 @@ jit_lock_reg_in_insn(JitCompContext *cc, JitInsn *the_insn, JitReg reg_to_lock)
         jit_set_last_error(cc, "generate insn failed");
     return ret;
 }
+
+#if WASM_ENABLE_FAST_JIT_DUMP != 0
+void
+jit_insn_add_comments(JitInsn *insn, const char *formatter, ...)
```

Comment:
Had better change `formatter` to `format`, like printf/snprintf:
int printf(const char *format, ...);
int snprintf(char *str, size_t size, const char *format, ...);